onnxruntime/contrib_ops/js/js_contrib_kernels.cc

```diff
@@ -11,6 +11,8 @@ class ONNX_OPERATOR_KERNEL_CLASS_NAME(kJsExecutionProvider, kMSDomain, 1, Gelu);
 class ONNX_OPERATOR_KERNEL_CLASS_NAME(kJsExecutionProvider, kMSDomain, 1, BiasSplitGelu);
 class ONNX_OPERATOR_KERNEL_CLASS_NAME(kJsExecutionProvider, kMSDomain, 1, BiasAdd);
 class ONNX_OPERATOR_KERNEL_CLASS_NAME(kJsExecutionProvider, kMSDomain, 1, SkipLayerNormalization);
+class ONNX_OPERATOR_KERNEL_CLASS_NAME(kJsExecutionProvider, kMSDomain, 1, FusedConv);
+class ONNX_OPERATOR_KERNEL_CLASS_NAME(kJsExecutionProvider, kMSDomain, 1, FusedConvTranspose);
```

Comment:
Not quite understanding this.

When the JS EP takes a Conv or ConvTranspose node it gets converted to the internal NHWC domain during partitioning. When the level 2 optimizer fuses it (after partitioning) I expected the node to remain in the internal NHWC domain so it's clear what the layout is.

Given that, when does a FusedConv{Transpose} in the contrib domain (which would be NCHW format) come into play for the JS EP?

Reply:
We have a choice from the frontend to set the preference, either channel-last (NHWC) or channel-first(NCHW). If the preference is set to channel-first then we could stil apply fusion.